### PR TITLE
Add aarch64 to the build targets

### PR DESCRIPTION
**Pull request by illyohs**
This patch adds the meson files necessary to build for aarch64

### CODE REVIEW

**Comment by keith-packard** [comment]
Thanks for adding aarch64 build instructions!

Given that there's a compiler for this available in Debian, I need to have the github action stuff updated to build the library so that your PR will be tested automagically. let me know if you have trouble understanding how that works; I'm certainly willing to help out, but the results need to be in your branch so that the tests will be run before I merge. All you should need to do is add the packages necessary to .github/Dockerfile and add a stanza in .github/workflows/test.yml to build. Running the tests is currently disabled as there's a bug in Debian qemu causing it to fail under docker, but when that's fixed, I'll re-enable them.

I'd really like to be able to test the library using qemu just like we can for arm and risc-v architectures. And that means being able to build complete applications using semihosting and some bare-metal startup code. I took a stab at this and have pushed the results to the aarch64 branch on top of your changes. I got stuck when trying to run one of the tests under qemu using the run-aarch64 script (provided in the patches) and decided to let you figure out what you wanted to do next.

In short, please update your branch to include the github action stuff so we can see it get compiled. In addition, if you're interested, I'd love to have the tests able to run under qemu (which works great when not run under docker for arm and risc-v), but I won't hold up the merge waiting for that as it may be more work than you're able to commit to right away.

**Comment by illyohs** [comment]
I'll work getting tests working in qemu, I have another PR for aarch64 for bare-metal in the works so I could kill two birds with one stone

**Comment by illyohs** [comment]
I'll work getting tests working in qemu, I have another PR for aarch64 for bare-metal in the works so I could kill two birds with one stone

**Comment by keith-packard** [comment]
Thanks for getting the github testing running. I've merged this to master.
